marketplace/deployer_helm_tiller_base/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM launcher.gcr.io/google/debian9
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    bash \
+    ca-certificates \
+    curl \
+    gettext \
+    git \
+    jq \
+    procps \
+    python \
+    python-pip \
+    python-setuptools \
+    python-yaml \
+    wget \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN pip install wheel \
+    && pip install google-cloud-storage
+
+RUN wget -q -O /bin/kubectl \
+        https://storage.googleapis.com/kubernetes-release/release/v1.9.0/bin/linux/amd64/kubectl \
+    && chmod 755 /bin/kubectl
+
+RUN mkdir -p /bin/helm-downloaded \
+    && wget -q -O /bin/helm-downloaded/helm.tar.gz \
+        https://storage.googleapis.com/kubernetes-helm/helm-v2.10.0-linux-amd64.tar.gz \
+    && tar -zxvf /bin/helm-downloaded/helm.tar.gz -C /bin/helm-downloaded \
+    && mv /bin/helm-downloaded/linux-amd64/helm /bin/ \
+    && rm -rf /bin/helm-downloaded \
+    && helm init --client-only \
+    && helm plugin install https://github.com/rimusz/helm-tiller \
+    && helm plugin install https://github.com/nouney/helm-gcs \
+    && helm repo add cloud-marketplace-tools gs://cloud-marketplace-tools-charts/
```

Comment:
https://github.com/helm/helm/blob/master/docs/chart_repository.md#google-cloud-storage

This can simply be: https://cloud-marketplace-tools.storage.googleapis.com/charts

Reply:
Done.